src/controller/Controller.hs

```diff
@@ -11,5 +14,29 @@ input :: Event -> GameState -> IO GameState
 input _ = return
```

Comment:
Dit is de functie die uiteindelijk naar de playIO in Main gaat, dus deze moet ook nog aangepast worden. Gezien de types van deze en jou nieuwe funtie kun je denk ik gewoon `return` aanroepen op het resultaat van `keyboardInputHandler`.